src/App.tsx

```diff
@@ -144,12 +144,33 @@ const AppNavigator = StackNavigator(Scenes,
     },
 );
 
+const WelcomeNavigator = StackNavigator({
+    IdentitySettingsContainer: {
+        screen: IdentitySettingsContainer,
+    },
+}, {
+    mode: 'card',
+    navigationOptions: {
+        header: null,
+    },
+});
+
+const SwNavigator = SwitchNavigator({
```

Comment:
I think MainNavigator or InitialNavigator would be a better name for this

Reply:
solved